packages/svelte/src/compiler/phases/3-transform/client/visitors/javascript-runes.js

```diff
@@ -315,7 +315,17 @@ export const javascript_visitors_runes = {
 						)
 					);
 					for (let i = 0; i < bindings.length; i++) {
-						bindings[i].expression = b.member(b.call('$.get', b.id(id)), b.literal(i), true);
+						const binding = bindings[i];
+						declarations.push(
+							b.declarator(
+								binding.node,
+								b.call(
+									'$.derived',
+									b.thunk(b.member(b.call('$.get', b.id(id)), b.literal(i), true))
+								)
+							)
+						);
+						binding.expression = b.call('$.get', binding.node);
```

Comment:
Can we omit this now? I'm not sure, but maybe the binding is marked being a derived so that `$.get` transformation happens automatically.